src/MyParcelComApiInterface.php

```diff
@@ -41,11 +41,10 @@ public function authenticate(AuthenticatorInterface $authenticator);
      * Get an array with all the regions, optionally regions can be filtered by
      * country code and region code.
      *
-     * @param string|null $countryCode
-     * @param string|null $regionCode
+     * @param mixed $filters
```

Comment:
I would recommend changing this to `@param array $filters` so people can see that it should be an array. It is also recommended to have the default value be `[]`, as `null` doesn't actually signify anything in this case (as in we do not need to differentiate between no filters set and an empty array of filters) and it's good practice not to mix types where you can avoid it.

Reply:
Yeah, I wasn't really sure about doing that as we want to be backward compatible. But I guess we could just set the docblock to state `array` and leave the actual parameter un-type-hinted (which we actually can't do because of php 5 anyway).

Reply:
You can typehint `array` in php 5, but yeah I didn't mean adding it as a typehint.